src/app/view/meter-read/add-bulk-reads/add-bulk-reads.component.html

```diff
@@ -0,0 +1,162 @@
+<div class="body-bg ">
+  <mat-card *ngIf="!showmeterreadinfo">
+    <mat-card-header
+      class="d-flex justify-content-between mat-crd-hide align-items-center"
+    >
+      <div>
+        <h2 class="pb-0 mb-0 fw-bold text-capitalize">
+          MeterRead Bulk Registration
+        </h2>
+      </div>
+      <div>
+        <button
+          class="btn btn-dark justify-content-end px-4"
+          (click)="JobDisplayList()"
+        >
+          <mat-icon class="align-middle">refresh</mat-icon>
+        </button>
+      </div>
+    </mat-card-header>
+    <hr />
+    <mat-card-content>
+      <form>
+        <div class="example-container1 row"></div>
+        <div class="example-container1 row">
+          <div
+            class="col-md-3"
+            *ngIf="loginuser.role === 'Admin' || loginuser.role === 'ApiUser'"
+          >
+            <mat-form-field appearance="outline">
+              <mat-label class="text-capitalize">Organization</mat-label>
+
+              <input
+                matInput
+                [(ngModel)]="orgname"
+                [ngModelOptions]="{ standalone: true }"
+                [matAutocomplete]="orgAuto"
+                (input)="filterOrgList()"
+              />
+
+              <mat-autocomplete
+                #orgAuto="matAutocomplete"
+                (optionSelected)="selectOrg($event)"
+              >
+                <mat-option
+                  *ngFor="let code of filteredOrgList"
+                  [value]="code.name"
+                >
+                  {{ code.name }}
+                </mat-option>
+              </mat-autocomplete>
+            </mat-form-field>
+          </div>
+          <div class="col-md-3">
+            <mat-form-field appearance="outline">
+              <div>
+                <mat-toolbar>
+                  <input
+                    matInput
+                    [value]="fileName"
+                    (click)="openFileExplorer()"
+                  />
+                </mat-toolbar>
+                <div class="d-flex justify-content-center">
+                  <input
+                    type="file"
+                    hidden
+                    id="fileInput"
+                    (change)="selectFile($event)"
+                    name="fileInput"
+                    accept=".csv"
+                  />
+                </div>
+              </div>
+            </mat-form-field>
+          </div>
+          <div class="col-md-6 upload-reset-buttons">
+            <button
+              class="btn btn-primary"
+              [disabled]="!currentFile"
+              (click)="upload()"
+            >
+              Upload</button
+            >&nbsp;
```

Comment:
You can remove the `&nbsp` and use a display layout like flex to put a gap between those buttons